KILTprotocol/sdk-js
# Dn api 175 public req attes met

## fixes #175 

## How to test:

Check docs:
Run npm run build:docs. This generates the doc at sdk-js/docs/api.
Open the so-generated files in your browser: sdk-js/docs/api/classes/requestforattestation.requestforattestation-1.html


## Checklist:
